user_tools/src/spark_rapids_pytools/cloud_api/databricks_azure.py

```diff
@@ -51,8 +52,16 @@ def _construct_cli_object(self) -> CMDDriverBase:
     def _install_storage_driver(self):
         self.storage = AzureStorageDriver(self.cli)
 
-    def _construct_cluster_from_props(self, cluster: str, props: str = None):
-        return DatabricksAzureCluster(self).set_connection(cluster_id=cluster, props=props)
+    def _construct_cluster_from_props(self, cluster: str, props: str = None, is_inferred: bool = False):
+        return DatabricksAzureCluster(self, is_inferred=is_inferred).set_connection(cluster_id=cluster, props=props)
+
+    def _construct_cluster_config(self, cluster_info: dict, default_config: dict):
+        cluster_conf = default_config
+        cluster_conf['executors'] = [{'node_id': '1234567890'} for _ in range(cluster_info['num_executor_nodes'])]
+        cluster_conf['driver_node_type_id'] = cluster_info['driver_instance']
+        cluster_conf['node_type_id'] = cluster_info['executor_instance']
+        cluster_conf['num_workers'] = cluster_info['num_executor_nodes']
```

Comment:
I remember that we discussed that before but probably we forgot to followup on the core side.
It is better if we can use camel case to be standard with all other files.
I understand that since 24.02.0 is being released; then the change in user-tools will cause crashes., but it is worth to get that done correctly in the first time.

Reply:
Will rename the fields in `cluster_info` dictionary to camel case.